backend/src/main/java/com/example/skicurort/curort/CurortControler.java

```diff
@@ -0,0 +1,57 @@
+package com.example.skicurort.curort;
+
+import static com.example.skicurort.curort.CurortMapper.*;
+
+import com.example.skicurort.exception.NoIdException;
+import java.util.List;
+import lombok.RequiredArgsConstructor;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+@RestController
+@RequiredArgsConstructor
+@RequestMapping("/api/curort")
+public class CurortControler {
+
+  private final CurortService curortService;
+
+  @GetMapping("/")
+  List<CurortDTO> getAll() {
+    return mapCurortsToCurortDTOs(curortService.getAll());
```

Comment:
Encje nie powinny wyciekać do controlera. Serwisy powinny używać w api DTO'sów. Encja jest wewnętrzną sprawą danej dziedziny. Encje są w domenie a DTO na styku domeny iświata, że tak powiem zew.

Reply:
Chodzi o to by wyszukiwanie encji i zmiana na dto była w serwisie a do controlera tylko i wyłacznie jak już trafiały dto?

Reply:
Tak dokładnie o to

Reply:
zrobione